packages/search-in-workspace/src/browser/search-in-workspace-result-tree-widget.tsx

```diff
@@ -586,8 +592,12 @@ export class SearchInWorkspaceResultTreeWidget extends TreeWidget {
     }
 
     protected renderResultLineNode(node: SearchInWorkspaceResultLineNode): React.ReactNode {
+        const lineNumber = this.searchInWorkspacePreferences['search.lineNumbers'] ? node.line : '';
         const prefix = node.character > 26 ? '... ' : '';
         return <div className={`resultLine noWrapInfo ${node.selected ? 'selected' : ''}`}>
+            <span className='siw-lineNumber'>
```

Comment:
btw if you want to skip conditionally element rendering completely:
```ts
this.searchInWorkspacePreferences['search.lineNumbers'] && <span className='siw-lineNumber'>...
```

Reply:
thank you!